app/src/main/java/org/fossasia/openevent/app/common/utils/ui/BackPressHandler.java

```diff
@@ -14,12 +16,23 @@
     public BackPressHandler() { }
 
     public void onBackPressed(Activity activity, Runnable action) {
-        if (backPressed + BACK_PRESS_RESET_TIME > System.currentTimeMillis()) {
-            action.run();
-        } else {
-            Toast.makeText(activity, "Press again to exit", Toast.LENGTH_SHORT).show();
+        if(activity instanceof MainActivity){
```

Comment:
Checking instance is bad. Java is statically typed language. This is dynamic behaviour. Always prefer static behaviour when possible and it is possible here